include/swift/Runtime/Config.h

```diff
@@ -216,6 +216,7 @@
 // But some of the runtime entries are invoked directly from
 // the foundation. Therefore they should be visible.
 #define SWIFT_RT_ENTRY_VISIBILITY SWIFT_RUNTIME_EXPORT
+#define SWIFT_EXTERN_RT_ENTRY_VISIBILITY SWIFT_EXTERN_RUNTIME_EXPORT
```

Comment:
I think we want this:
1. `SWIFT_RUNTIME_EXPORT` has both `extern "C"` and `__declspec` as appropriate
2. every export has a header declaration
3. header declarations of exports all use `SWIFT_RUNTIME_EXPORT`
4. non-header definitions do not use `SWIFT_RUNTIME_EXPORT` or `extern "C"` or `__declspec`.